Assignment8/Assignment8/Assignment8/MainWindow.xaml.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Microsoft.Win32;
+using System;
+using System.IO;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Threading;
+
+namespace Assignment8
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private readonly DispatcherTimer dispatcherTimer = new();
+        
+        private DateTime _StartTime = DateTime.Now;
+        private string _Description = null!;
+        private bool _IsStopped = true;
+        private int entryNumber = 1;
+
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            dispatcherTimer.Tick += new EventHandler(DispatcherTimer_Tick!);
```

Comment:
Event handler can be simplified.
```suggestion
            dispatcherTimer.Tick += DispatcherTimer_Tick!;
```